### ISSUE TITLE
"An exception has occurred" message after trying to update the game 

### ISSUE THREAD

**Issue by Cryptwrath** [Monika-After-Story/MonikaModDev#10106]
This is what was shown on screen, need help please 

While running game code:
  File "game/sprite-chart.rpy", line 2324, in script
  File "game/sprite-chart.rpy", line 2332, in <module>
  File "game/sprite-chart.rpy", line 3966, in MASMonika
TypeError: deprecated() takes exactly 0 arguments (1 given)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/sprite-chart.rpy", line 2324, in script
  File "C:\Users\asus\OneDrive\Desktop\DDLC-1.1.1-pc\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\asus\OneDrive\Desktop\DDLC-1.1.1-pc\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/sprite-chart.rpy", line 2332, in <module>
  File "game/sprite-chart.rpy", line 3966, in MASMonika
TypeError: deprecated() takes exactly 0 arguments (1 given)

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.15

edit: I should add that this occurred after i tried to update the game and the update failed, prompting me to restart it. When I did so, I received the message that you are seeing now.

**Comment by dreamscached** [comment]
Duplicate of #10104.
